src/sentry/api/endpoints/project_platforms.py

```diff
@@ -2,17 +2,24 @@
 
 from rest_framework.response import Response
 from sentry.api.bases.project import ProjectEndpoint
-from sentry.api.serializers import serialize, register, Serializer
 from sentry.models import ProjectPlatform
+from rest_framework import serializers, status
+from sentry.api.serializers import serialize
+from django.core.exceptions import ObjectDoesNotExist
 
 
-@register(ProjectPlatform)
-class ProjectPlatformSerializer(Serializer):
-    def serialize(self, obj, attrs, user):
-        return {'platform': obj.platform, 'dateCreated': obj.date_added}
+class ProjectPlatformSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ProjectPlatform
+        fields = ('platform',)
```

Comment:
@mattrobenolt 
Should I get rid of the model serializer since we aren't using it really?

Reply:
Yeah, since we're not longer POSTing new ones, we don't need this serializer at all.